tests/test_server_behavior.py

```
@@ -0,0 +1,545 @@
+    async def test_pool_best_strategy_selects_fastest_proxies(self, user_pool):
+        """Test that 'best' strategy selects proxies with best response times.
+        
+        Users configure strategy='best' to get fastest proxies.
+        """
+        # Get multiple proxies and verify they're selected by performance
+        proxies = []
+        for _ in range(3):
+            try:
+                proxy = await user_pool.get("HTTP")
+                proxies.append(proxy)
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)